auth_mellon_config.c

```diff
@@ -1099,6 +1099,14 @@ const command_rec auth_mellon_commands[] = {
         " '/'."
         ),
     AP_INIT_TAKE1(
+        "MellonCookieSameSite",
+        ap_set_string_slot,
```

Comment:
I have created a separate pull request with the amended parser.

As a result, this pull request can be closed and it's up to you whether to include this feature.

Regards
Vittal